src/main/java/com/powsybl/openloadflow/network/LoadFlowType.java

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.openloadflow.network;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public enum LoadFlowType {
```

Comment:
I'm ok with that name, but just had a look to the literature anyway, for instance [this](https://ieeexplore.ieee.org/abstract/document/1451474) and [that](https://ieeexplore.ieee.org/abstract/document/4956966). Maybe you'd be more convinced with `LoadFlowModel` or `LoadFlowMethod`?

Reply:
I really like power flow model!

Reply:
"method" is more like the way to solve you model